src/autopas/tuning/selectors/TraversalSelector.h

```diff
@@ -11,9 +11,12 @@
 #include <unordered_map>
 #include <vector>
 
+#include "autopas/baseFunctors/TriwiseFunctor.h"
```

Comment:
This include is not needed if I see this right because whenever you use `TriwiseFunctor` it is a template.